src/groovy/org/pih/warehouse/importer/OutboundStockMovementExcelImporter.groovy

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2012 Partners In Health.  All rights reserved.
+ * The use and distribution terms for this software are covered by the
+ * Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+ * which can be found in the file epl-v10.html at the root of this distribution.
+ * By using this software in any fashion, you are agreeing to be bound by
+ * the terms of this license.
+ * You must not remove this notice, or any other, from this software.
+ **/
+package org.pih.warehouse.importer
+
+import org.codehaus.groovy.grails.commons.ApplicationHolder
+import org.grails.plugins.excelimport.ExcelImportUtils
+
+class OutboundStockMovementExcelImporter extends AbstractExcelImporter {
+    static Map columnMap = [
+        sheet   : 'Sheet1',
+            startRow : 1,
+            columnMap : [
+                    'A': 'Source',
```

Comment:
I think this should be mapped into proper code instead of label value (same in the propertyMap)